src/test/java/org/candlepin/subscriptions/metering/OpenShiftMetricsHelper.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2009 - 2019 Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.subscriptions.metering;
+
+import org.candlepin.subscriptions.prometheus.model.QueryResult;
+import org.candlepin.subscriptions.prometheus.model.QueryResultData;
+import org.candlepin.subscriptions.prometheus.model.QueryResultDataResult;
+import org.candlepin.subscriptions.prometheus.model.ResultType;
+import org.candlepin.subscriptions.prometheus.model.StatusType;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+/**
+ * A utility class to help with creating prometheus API response data.
+ */
+public class OpenShiftMetricsHelper {
+
+    private OpenShiftMetricsHelper() {
+
+    }
+
+    public static QueryResult buildSubscriptionLabelsQueryResult(String account, String clusterId,
+        String serviceLevel, String usage, List<List<BigDecimal>> timeValueTuples) {
+        QueryResultDataResult dataResult = buildLabelDataResult(account, clusterId, serviceLevel,
+            usage, timeValueTuples);
+
+        return new QueryResult()
+            .status(StatusType.SUCCESS)
+            .data(
+               new QueryResultData()
+                   .resultType(ResultType.MATRIX)
+                   .addResultItem(dataResult)
+            );
+    }
+
+    public static QueryResult buildCoresMetricQueryResult(String account, String clusterId,
+        List<List<BigDecimal>> timeValueTuples) {
+        QueryResultDataResult dataResult = new QueryResultDataResult()
+            .putMetricItem("_id", clusterId)
```

Comment:
CTRL+F on this page has seven results of `"_id"`...I know at least some of them are probably duplicates because of the diffs....but is it worth making this a constant field?